go/batch-submitter/Makefile

```diff
@@ -7,6 +7,9 @@ LDFLAGSSTRING +=-X main.GitDate=$(GITDATE)
 LDFLAGSSTRING +=-X main.GitVersion=$(GITVERSION)
 LDFLAGS := -ldflags "$(LDFLAGSSTRING)"
 
+CTC_DEPLOYMENT := ../../packages/contracts/deployments/mainnet/CanonicalTransactionChain.json
```

Comment:
There's a chance that this will limit us in the future if we need to test out changes to these contracts that would result in a different ABI. Would it be possible to get this from `packages/contracts/artifacts/` instead?

Reply:
great suggestion, thanks!